### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MMOsei**
I would like to merge the Dev branch to master. Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+              <li><a href="../index.html">Home</a></li>
+              <li><a href="pages/about.html">About</a></li>
+              <li><a href="pages/contact.html">Contact Us</a></li>
+            </ul>
+          </div><!-- /.navbar-collapse -->
+        </div><!-- /.container-fluid -->
+      </nav>
+
+    <!-- jumbatron -->
+    <div class="overlay jumbotron landing-jt">
```
Comment (ACKAdeyemi) [inline index.html]: Indent this <div> tag and everything below it down to line 90

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      <div class="container">
+        <h1>We are Spartans!</h1>
+        <!-- <img src="Images/sparta-global-logo-white.svg" alt="Spart Global Logo"> -->
+        <p></p>
```
Comment (ACKAdeyemi) [inline index.html]: Remove this p tag.

**Review by ACKAdeyemi — CHANGES_REQUESTED** [review]
Website looks cool and has many great features, everything works (mostly), good comments but unfortunately the code isn't formatted well.

Go through all pages and indent correctly and consistently. Also delete unnecessary spaces between lines and be consistent. So either 1 space or no spaces between divs, css selectors, etc.

Remove unnecessary commented lines of code e.g. lines where you were experimenting, lines that don't work, etc.

Remove empty tags. 

index issues:
- Remove p tag on line 50
- Logo doesn't link to home (logo doesn't link to home on any of the pages)

Get that sorted and should be fine.

Could add a couple more explanatory comments in your css doc.

**Comment by MMOsei** [comment]
Hi @ACKAdeyemi @isabelvalente ,

All changes made, as requested, and ready for dev to be merged to master.

Ready for review.

Many thanks,

Mimi

**Review by ACKAdeyemi — APPROVED** [review]

**Pull request merged**